docs/src/dummy.stories.js

```diff
@@ -1,15 +1,23 @@
 import HelloWorld from '@wmde/wikit-vue-components/src/components/HelloWorld';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { withA11y } from '@storybook/addon-a11y';
 
 export default {
 	title: 'Components/Dummy',
+	decorators: [ withKnobs, withA11y ],
```

Comment:
I assume we want this to be true for all (component) stories.

Was the decision not to register those decorators globally, 
[e.g. like was done in bridge](https://gerrit.wikimedia.org/r/plugins/gitiles/mediawiki/extensions/Wikibase/+/5f1d7d1/client/data-bridge/.storybook/preview.js), a conscious one? What were the arguments?